app/views/layouts/_center_div_no_listnav.html.haml

```diff
@@ -8,7 +8,7 @@
         .row.toolbar-pf#toolbar
           .col-sm-12
             = render :partial => "layouts/angular/toolbar"
-      .row#main-content.miq-layout-center_div_no_listnav{:class => miq_layout_center_div_no_listnav_class }
+      .row#main-content.miq-layout-center_div_no_listnav{:class => @lastaction == "show_dashboard" || @layout == "monitor_alerts_overview" ? 'miq-body' : ''}
```

Comment:
Does this change belong here? (and the removal of `miq_layout_center_div_no_listnav_class`?)

It seems like the logic changes from:

```
(!@in_a_form && (@lastaction == "show_dashboard" || @layout == "monitor_alerts_overview")) || show_list_with_no_provider?
```

and now the `show_list_with_no_provider?` bit and the `!@in_a_form` bits are gone.

Reply:
It used to be this way before https://github.com/ManageIQ/manageiq-ui-classic/pull/5455,
so I guess the `! @in_a_form` bit could have been only relevant for that.

But then https://github.com/ManageIQ/manageiq-ui-classic/pull/5434 added the no provider logic, not sure how it affects that?